src/amy.c

```diff
@@ -163,8 +257,8 @@ void add_delta_to_queue(struct delta d) {
         event_counter++;
 
     } else {
-        // If there's no room in the queue, just skip the message
```

Comment:
Oh, I think this is happening when Xanadu gets too backed up, or when the keyboard hits too many notes too fast.  Was wondering how we were dropping messages.